### DISCUSSION TITLE
pyodide doesn't take the local linux file systems

### DISCUSSION THREAD

**Discussion by bommithis** [pyscript/pyscript#1325] [Q&A]
pyodide by default writes to /home/pyodide; is there no way to redirect it to /tmp or any other file system in unix/linux.??

**Comment by JeffersGlass** [comment]
When you say 'writes', what do you mean? Are you opening a file handle within Python and writing to it? If so, you can reference any other filepath in the virtual filesystem just like in "desktop" Python.
